### DISCUSSION TITLE
Editor Updates?

### DISCUSSION THREAD

**Discussion by RetrogradeDev** [ScratchAddons/ScratchAddons#6217] [Help] [answered]
Hello, I saw that there were bug fixes for a scratch update, but I couldn't find more about that update anywhere. Can anyone tell me what is going to change?

**Comment by pufferfish101007** [comment, marked as answer]
https://scratchfoundation.github.io/scratch-gui/beta

purple navbar and optional high contrast blocks

**Reply by RetrogradeDev** [reply]
No!! Scratch needs to be blue!

**Reply by BroJac5246** [reply]
> No!! Scratch needs to be blue!

🤷 Scratch Addons can fix that :D

**Reply by RetrogradeDev** [reply]
I know, but I feel sorry for those who don't have SA  ;-)

**Reply by ghost** [reply]
I think scratch will stay blue (edit: ho ok it will be purple my bad), what's new is the "high contrast blocs" (who is something alomst no one wants... I'm so disapointed... I think scratchteam must create pools to know what people want. They often reicive bad reviews when they share updates because they don't do what people want at all...)

**Reply by CST1229** [reply]
> I think scratch will stay blue, it's purple because it's the beta version

No, the Scratch editor [will become purple](https://github.com/scratchfoundation/scratch-gui/pull/8858) (it was changed for text contrast).

**Reply by mxmou** [reply]
> it was changed for text contrast

(which doesn't make much sense because a darker shade of blue - for example `$motion-tertiary`, `#3373cc` - would also have enough contrast.)

**Reply by Secret-chest** [reply]
> > it was changed for text contrast
> 
> (which doesn't make much sense because a darker shade of blue - for example `$motion-tertiary`, `#3373cc` - would also have enough contrast.)

It was changed for lil' girls
